### PR TITLE
Zoom to bounds of community

### LINKED ISSUE #31: Community map should be zoomed in a bit
When using the desktop to see the community map the world is duplicated creating the feeling that there are places where people are not contributing when it not true.
I'm guessing there's a need to do `fitBounds` so that the world would fit for the relevant device?
P.S. Thanks a lot for all the rewrite done lately!! :-)
image

### PR DESCRIPTION
**Pull request by maxammann**
fixes #31

### CODE REVIEW

**Review by HarelM — APPROVED** [review]

**Comment by wipfli** [comment]
Thanks Max. Feel free to merge if you get one approval from someone with write access. This change looks good to me!

**Pull request merged**
